source/ada/lsp-ada_handlers.adb

```diff
@@ -132,6 +132,13 @@ package body LSP.Ada_Handlers is
    --  Reload as project source dirs the directories in
    --  Self.Project_Dirs_Loaded.
 
+   function Get_Reference_Kind
```

Comment:
There can be many "reference kinds", but this subprogram only checks the nature of "calls". So I would suggest renaming this to `Get_Call_Reference_Kind` and, in the doc, say that this works on Nodes that represent subprogram calls.